ui/app/AppLayouts/Chat/panels/communities/CommunityProfilePopupInviteMessagePanel.qml

```diff
@@ -32,14 +33,27 @@ ColumnLayout {
 
     StatusInput {
         id: messageInput
-        label: qsTr("Invitation Message")
```

Comment:
I believe the former is correct; titles and headers should always be in title case, despite what the design shows

Reply:
Reverted